### ISSUE TITLE
Ship Windows GPU libtensorflow builds

### ISSUE THREAD

**Issue by nkreeger** [tensorflow/tfjs#602]
Windows + CUDA is proving hard to build on at least two Win10 machines. This issue is a placeholder to track this integration.

**Comment by ashbeats** [comment]
On win 10 and did not face any issues installing `npm install @tensorflow/tfjs-node-gpu`

**Comment by iacovlev-pavel** [comment]
Installing `@tensorflow/tfjs-node-gpu` works, but when running the `mnist-node` example only the CPU is used.
On the same machine python `tensorflow-gpu` works and use the NVIDIA GPU.

Note that for the `mnist-node` example the following changes were made:
```
npm install @tensorflow/tfjs-node-gpu
Replace require('@tensorflow/tfjs-node') with require('@tensorflow/tfjs-node-gpu'); in main.js
```

**Comment by nkreeger** [comment]
Yep - that's because we don't have a GPU build yet. https://github.com/tensorflow/tfjs-node/blob/master/scripts/get_libtensorflow.js#L37 That script will default to WINDOWS CPU until we have a library we can distribute. I'll update this ticket once I have that building.

**Comment by wookoouk** [comment]
Is it possible to manually build it for GPU?

**Comment by nkreeger** [comment]
We just shipped 0.1.14 with windows support for @tensorflow/tfjs-node-gpu
